### PR TITLE
Add more model numbers

### PR DESCRIPTION
**Pull request by kevdagoat**
Added into the supported list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,11 @@
   <ul>
   <li>DGA4132</li>
   <li>DGA4130</li>
+  <li>TG789vac v3</li>
```
Comment (FrancYescO) [inline README.md]: does this naming even exist ?

File: README.md [hunk 2]
```diff
@@ -2,8 +2,11 @@
...
   <li>TG789vac v2</li>
   <li>TG799vac</li>
+  <li>TG799vac XTREAM</li>
+  <li>TG799vac XTREAM 35b</li>
```
Comment (FrancYescO) [inline README.md]: the Fastweb's one is called G7**8**9vac XTREAM 35**B**

**Comment by FrancYescO** [comment]
I think due the the confusing Technicolor/ISPs naming we should start grouping also by board naming
ie 
DGA4132 / vbnt-s
DGA4131 / vbnt-o_fastweb
TG789vac / vant-6 / vant-f

**Comment by kevdagoat** [comment]
Ok. Yes the v3 does exist (I think it is vbnt-s)

File: README.md (hunk 1 above)
Comment (kevdagoat) [reply README.md]: Yep

File: README.md (hunk 2 above)
Comment (kevdagoat) [reply README.md]: Ok will edit that

**Comment by kevdagoat** [comment]
Have a look here for firmware and model numbers https://whirlpool.net.au/wiki/hack_technicolor#hw_model_1622

**Pull request merged**